data/meson.build

```diff
@@ -30,25 +30,29 @@ if get_option('daemon')
   )
 endif
 
-con2 = configuration_data()
-con2.set('libexecdir', libexecdir)
-con2.set('bindir', bindir)
-con2.set('localstatedir', localstatedir)
-con2.set('datadir', datadir)
-con2.set('sysconfdir', default_sysconfdir)
+if get_option('systemd')
+  con2 = configuration_data()
+  con2.set('libexecdir', libexecdir)
+  con2.set('bindir', bindir)
+  con2.set('datadir', datadir)
 
-# replace @libexecdir@
-configure_file(
-  input : 'org.freedesktop.fwupd.service.in',
-  output : 'org.freedesktop.fwupd.service',
-  configuration : con2,
-  install: true,
-  install_dir: join_paths(datadir,
-                          'dbus-1',
-                          'system-services'),
-)
+  if systemd.version().version_compare('>= 231')
```

Comment:
So, before 231 we couldn't even include an empty ReadWritePaths= in the service file?

Reply:
You'll end up with something like this:
`Unknown lvalue 'ReadWritePaths' in section 'Service'`

I think this was actually a case of the option being renamed though.  So maybe the right thing to do is on `<231` use:
`
ReadWriteDirectories=
`